src/interaction/interactive_api.jl

```diff
@@ -126,12 +140,23 @@ Return all `(plot, index)` pairs in a `(xy .- range, xy .+ range)` region
 sorted by distance to `xy`. See [`pick`](@ref) for more details.
 """
 function pick_sorted(scene::Scene, xy, range)
+    PICK_TRACKING[] && (_PICK_COUNTER[] += 1)
     screen = getscreen(scene)
     screen === nothing && return Tuple{AbstractPlot, Int}[]
+    if PICK_TRACKING[]
+        # if the Makie implementation is used we'd double count if we just increment here
```

Comment:
Maybe we should let the backend do this and just have one counter in native_picking?

Reply:
I generally prefer testing things that don't require a backend without the backend. I don't have a strong reason for it though. I guess it helps balance out test times so a full CI run doesn't get longer?
Backends also have multiple methods that would need tracking (point, rect, maybe closest, sorted) and we'd have to duplicate code across backends if we added it there.